content/master/guides/mrd-activation-policies.md

```diff
@@ -0,0 +1,689 @@
+---
+title: Writing MRD activation policies
+weight: 150
+description: Learn how to create effective activation policies for managed resources
+---
```

Comment:
I'd be comfortable dropping this guide entirely. If there's anything important that's only mentioned in this guide right now, it could go in the adjacent "using MRDs and activation policies" guide.